src/settings/settingsManager.ts

```diff
@@ -47,6 +49,10 @@ export class SettingsManager {
 
   public async setIsoPath(isoPath: string | null): Promise<void> {
     await this._set("settings.isoPath", isoPath);
+    if (isoPath) {
+      const gameDir = path.dirname(isoPath);
+      await addGamePathToInis(gameDir);
+    }
```

Comment:
Can you add this ipc handler logic in `src/settings/main.ts` instead? I feel like the settings manager should only have to worry about persisting and restoring from `electron-settings`.